### PR TITLE
Fixes #43.  Files is a list not a dict.  

### LINKED ISSUE #43: Values file that works with helm does not work with reckoner
This values file doesn't work with reckoner but works with a normal install:

```
prometheus:
  url: http://linkerd-prometheus.linkerd.svc

rules:
  default: false
  custom:
    - seriesQuery: 'response_latency_ms_bucket{namespace!="",pod!=""}'
      resources:
        template: '<<.Resource>>'
      name:
        matches: '^(.*)_bucket$'
        as: "${1}_50th"
      metricsQuery: 'histogram_quantile(0.50, sum(irate(<<.Series>>{<<.LabelMatchers>>, direction="inbound"}[5m])) by (le, <<.GroupBy>>))'

    - seriesQuery: 'response_latency_ms_bucket{namespace!="",pod!=""}'
      resources:
        template: '<<.Resource>>'
      name:
        matches: '^(.*)_bucket$'
        as: "${1}_95th"
      metricsQuery: 'histogram_quantile(0.95, sum(irate(<<.Series>>{<<.LabelMatchers>>, direction="inbound"}[5m])) by (le, <<.GroupBy>>))'

    - seriesQuery: 'response_latency_ms_bucket{namespace!="",pod!=""}'
      resources:
        template: '<<.Resource>>'
      name:
        matches: '^(.*)_bucket$'
        as: "${1}_99th"
      metricsQuery: 'histogram_quantile(0.99, sum(irate(<<.Series>>{<<.LabelMatchers>>, direction="inbound"}[5m])) by (le, <<.GroupBy>>))'
```

```
[ do-sfo2-reckoner-demo   ] [pw]  1m0s
✓ reckoner --log-level=DEBUG plot course.yaml  --heading prometheus-adapter
2019-01-08 10:24:28 root[64881]    DEBUG helm help
2019-01-08 10:24:28 root[64881]    DEBUG helm version --short --server
2019-01-08 10:24:29 root[64881]    DEBUG Installing repository: {'url': 'https://kubernetes-charts-incubator.storage.googleapis.com', 'name': 'incubator'}
2019-01-08 10:24:29 root[64881]    DEBUG helm repo list
2019-01-08 10:24:29 root[64881]    DEBUG Chart repository incubator already installed
2019-01-08 10:24:29 root[64881]    DEBUG Installing repository: {'url': 'https://kubernetes-charts.storage.googleapis.com', 'name': 'stable'}
2019-01-08 10:24:29 root[64881]    DEBUG helm repo list
2019-01-08 10:24:30 root[64881]    DEBUG Chart repository stable already installed
2019-01-08 10:24:30 root[64881]    DEBUG Installing repository: {'url': 'http://influx-charts.storage.googleapis.com', 'name': 'influx'}
2019-01-08 10:24:30 root[64881]    DEBUG helm repo list
2019-01-08 10:24:30 root[64881]    DEBUG Chart repository influx already installed
2019-01-08 10:24:30 root[64881]    DEBUG helm repo update
2019-01-08 10:24:32 root[64881]    DEBUG Helm Minimum Version is: 2.12.1
2019-01-08 10:24:32 root[64881]    DEBUG helm version --short --client
2019-01-08 10:24:32 root[64881]    DEBUG Helm Installed Version is 2.12.1
2019-01-08 10:24:32 root[64881]    DEBUG Reckoner Minimum Version is 0.10.1
2019-01-08 10:24:32 root[64881]    DEBUG Reckoner Installed Version is 0.10.3
2019-01-08 10:24:32 root[64881]    DEBUG helm version --short --client
2019-01-08 10:24:32 root[64881]     INFO Installing prometheus-adapter
2019-01-08 10:24:32 root[64881]    DEBUG helm repo list
2019-01-08 10:24:32 root[64881]    DEBUG Chart repository stable already installed
2019-01-08 10:24:32 root[64881]    DEBUG Updating chart dependencies: stable/prometheus-adapter
2019-01-08 10:24:32 root[64881]    ERROR Helm upgrade failed. Rolling back prometheus-adapter
2019-01-08 10:24:32 root[64881]    DEBUG Traceback (most recent call last):
  File "/Users/asuderma/repos/reactiveops/reckoner/reckoner/course.py", line 122, in plot
    chart.install(namespace=self.namespace, context=self.context)
  File "/Users/asuderma/repos/reactiveops/reckoner/reckoner/chart.py", line 214, in install
    for file in self.files:
  File "/Users/asuderma/repos/reactiveops/reckoner/reckoner/chart.py", line 87, in files
    return dict(self._chart.get('files', []))
ValueError: dictionary update sequence element #0 has length 31; 2 is required

2019-01-08 10:24:32 root[64881]    ERROR ERROR: Some charts failed to install and were rolled back
2019-01-08 10:24:32 root[64881]    ERROR  - prometheus-adapter
[ do-sfo2-reckoner-demo   ] [pw]
✓ helm delete --purge prometheus-adapter
Error: release: "prometheus-adapter" not found
[ do-sfo2-reckoner-demo   ] [pw]
 1 ✘ helm install -n^C
[ do-sfo2-reckoner-demo   ] [pw]
 130 ✘ helm install stable/prometheus-adapter --name prometheus-adapter -f prometheus-adapter.values.yml --namespace linkerd
NAME:   prometheus-adapter
LAST DEPLOYED: Tue Jan  8 10:25:08 2019
NAMESPACE: linkerd
STATUS: DEPLOYED

RESOURCES:
==> v1/ClusterRole
NAME                                 AGE
prometheus-adapter-server-resources  2s
prometheus-adapter-resource-reader   2s

==> v1/Service
NAME                TYPE       CLUSTER-IP      EXTERNAL-IP  PORT(S)  AGE
prometheus-adapter  ClusterIP  10.245.111.139  <none>       443/TCP  2s

==> v1/Deployment
NAME                DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
prometheus-adapter  1        1        1           0          2s

==> v1/ConfigMap
NAME                DATA  AGE
prometheus-adapter  1     2s

==> v1/ServiceAccount
NAME                SECRETS  AGE
prometheus-adapter  1        2s

==> v1/ClusterRoleBinding
NAME                                      AGE
prometheus-adapter:system:auth-delegator  2s
prometheus-adapter-resource-reader        2s
prometheus-adapter-hpa-controller         2s

==> v1/RoleBinding
NAME                            AGE
prometheus-adapter-auth-reader  2s

==> v1beta1/APIService
NAME                           AGE
v1beta1.custom.metrics.k8s.io  2s

==> v1/Pod(related)
NAME                                READY  STATUS             RESTARTS  AGE
prometheus-adapter-9876c8dc6-q8d6r  0/1    ContainerCreating  0         2s


NOTES:
prometheus-adapter has been deployed.
In a few minutes you should be able to list metrics using the following command:

  kubectl get --raw /apis/custom.metrics.k8s.io/v1beta1
```

### PR DESCRIPTION
**Pull request by sudermanjr**
Also don't need to raise the error, it is already logged.

### CODE REVIEW

File: reckoner/chart.py [hunk 1]
```diff
@@ -229,7 +230,6 @@ def install(self, namespace=None, context=None):
             logging.info(r.stdout)
         except HelmClientException, e:
             logging.error(e)
-            raise e
```
Comment (ejether) [inline reckoner/chart.py]: Did you remove the re-raise on purpose?

Comment (sudermanjr) [reply reckoner/chart.py]: Yes, it seemed somewhat unnecessary.  If it needs to be there please let me know

Comment (ejether) [reply reckoner/chart.py]: I _think_ the initial intent of re-raising was so you could log the error and then handle it gracefully at "top": https://github.com/reactiveops/reckoner/blob/master/reckoner/course.py#L119 but it doesn't look like that ever got finished. 

However, as it stands now, the error will be logged and then the post install hook on https://github.com/reactiveops/reckoner/blob/master/reckoner/course.py#234 will run even though the `Helm.Client().upgrade` threw a `HelmClientException`. That is probably not the intended behavior. 

Adding a `return` or `raise` will stop the post_install_hook from running.

Comment (endzyme) [reply reckoner/chart.py]: This sounds like a test case. #allHooksRunEvenIfInstallFails

**Pull request merged**